### ISSUE TITLE
Is it possible to release the training logs?

### ISSUE THREAD

**Issue by zc08** [qihao067/CrossFlow#17]
Hello @qihao067 : 
Thank you so much for sharing the code for your inspiring work! 

I am new to training text-to-image models and am working with limited GPU resources (4x RTX 5090s). As the training procedure takes a long time on my machine, I really want to ensure the training is going on as expected. It would be incredibly helpful to have access to the training logs.
E.g. to know typical loss values at different stages of training, or when the generated images begin to make sense (at a specific batch size and training step).

Could you please consider releasing the training logs? I believe this would be beneficial for me and many others to verify our training setups and better understand the model. Thank you again for your consideration and contribution to the field!

**Comment by qihao067** [comment]
Hello @zc08 ,

Thank you for your kind words! I completely understand how challenging it can be to train a text-to-image model, especially with limited GPU resources.

Unfortunately, I no longer have access to the full training logs. I only trained the base model from scratch once (due to limited GPU availability in our lab), and all released models are finetuned versions of that base. I do still have some logs from the finetuning phases, but they likely won’t be as helpful for your case.

However, I was able to find the sample images the model generated every 5K training steps when training from scratch. You can download them here:

https://huggingface.co/QHL067/CrossFlow/blob/main/samples.tar

In general:
- You can expect the model to start generating somewhat related images around 30K–40K steps
- More coherent and understandable results tend to appear around 50K steps

One tip I’d strongly recommend: even with limited resources, try to use the largest batch size you can manage. I think for this results, I use a batch size of 512, this significantly helps with training stability and convergence.

Hope this helps, and best of luck with your experiments!
